src/constants/Api/Orders/ordersApi.ts

```diff
@@ -1,9 +1,15 @@
 import { HTTP_METHODS } from 'src/helper/api'
 import Fetcher from 'src/services/Fetcher'
-import { OrderData, OrdersResponse } from './api'
+import { OrderData, OrdersInfoResponse, OrdersResponse } from './api'
 
 const fetcher = new Fetcher()
 
+export const requestOrdersInfo = (data: string) =>
+  fetcher.requestToReceive<OrderData, OrdersInfoResponse>({
+    url: `orders?${data}`,
```

Comment:
you do not need to configure link that has some query parameters, there is better place for that, think about it